pom.xml

```diff
@@ -71,6 +71,19 @@
 		        <version>${reportingVersion}</version>
 		        <scope>provided</scope>
 	        </dependency>
+            <dependency>
+                <groupId>org.openmrs.api-1.10</groupId>
```

Comment:
Darius suggested to add the api-1.10 dependencies so as to have access to the OrderFrequency handler rather than implement one myself. However, I guess I will have to remove it because I still couldn't can it to work.